content/api/api.go

```diff
@@ -0,0 +1,463 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+// Package api provides an api server for a content store.
```

Comment:
Could you add some documentation here to make it clear what this package is for?

Reply:
Do you mean we should provide some usecases?

My usecase is to allow `buildkitd` to send caches to `buildctl` via the content API over BuildKit session API, but I'm not sure I should mention my usecase here.